config/localization/en/app.json

```diff
@@ -1108,7 +1108,9 @@
       "STOP_LOSS_TOOLTIP": "Protects against losses by closing a trader’s position once the oracle price crosses the trigger price.",
       "TAKE_PROFIT_TOOLTIP": "Allows traders to set targets and protects profits on positions by specifying a price at which to close an open position for profit.",
       "TP_LIMIT": "TP Limit",
-      "TP_PRICE": "TP Price"
+      "TP_PRICE": "TP Price",
+      "TRIGGERS_INFO_AUTOMATICALLY_CANCELED": "Take profit and stop loss orders will automatically be canceled if the underlying position is closed.",
+      "TRIGGERS_INFO_CUSTOM_AMOUNT": "Configuring a custom amount will apply to both take profit and stop loss orders."
```

Comment:
nit, I think we capitalize Take Profit and Stop Loss generally - can we capitalize here? Same with above string